contribs/gnohealth/internal/timestamp/timestamp.go

```diff
@@ -35,7 +35,7 @@ func NewTimestampCmd(io commands.IO) *commands.Command {
 	return commands.NewCommand(
 		commands.Metadata{
 			Name:       "timestamp",
-			ShortUsage: "[flags]",
+			ShortUsage: "gnohealth timestamp [flags]",
```

Comment:
```suggestion
			ShortUsage: "timestamp [flags]",
```

We omit the command name in general

Reply:
Ok thanks 👍